db/migrate/20220809102256_add_collection_reference_to_hesa_students.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddCollectionReferenceToHesaStudents < ActiveRecord::Migration[6.1]
+  def change
+    add_column :hesa_students, :collection_reference, :string
```

Comment:
I'm not sure how this will be queried, but should we also consider indexing this col?

Reply:
I can't see this table been queried often as we enter the new academic year. We access records using `trainee.hesa_student`.